rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -55,8 +64,17 @@ include::modules/rosa-hcp-vpc-manual.adoc[leveloffset=+2]
 * link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-getting-started.html[Get Started with Amazon VPC]
 * link:https://developer.hashicorp.com/terraform[HashiCorp Terraform documentation]
 
-include::modules/rosa-hcp-byo-oidc.adoc[leveloffset=+1]
-//include::modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc[leveloffset=+2]
+[id="rosa-hcp-byo-odic-overview_{context}"]
+== Bring-Your-Own OpenID Connect Configuration
```

Comment:
I am not sure if the "bring your own" is precise
@gdbranco Would you help double confirm if we can use "Bring-Your-Own" or byo in doc?

Reply:
I'd say we should probably move external references to this as Unmanaged OIDC Configuration (when it is hosted in customers aws account) and Managed OIDC Configuration (when hosted by Red Hat)

@wgordon17 Do you have any insights on correct terms?

Reply:
@yingzhanredhat, good point. I revised this to say `Creating a managed OpenID Connect Configuration`. Then, the first level module is `Generating your own OpenID Connect configuration`. Does this work better?

CC: @gdbranco @wgordon17